### PR TITLE
Change the place which function adjustTextFieldPosition was called at

### PR DESCRIPTION
**Pull request by HoGarfield**
EditBox`s position will not be changed when parent`s position had be changed.So the position of CCEditBoxImplIOS still in the old position.

### CODE REVIEW

File: extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm [hunk 1]
```diff
@@ -607,6 +605,8 @@ static CGPoint convertDesignCoordToScreenCoord(const Point& designCoord, bool bI
 {
 	_label->setVisible(false);
 	_labelPlaceHolder->setVisible(false);
+    
+    this->adjustTextFieldPosition();
```
Comment (dumganhar) [inline extensions/GUI/CCEditBox/CCEditBoxImplIOS.mm:609]: You took `adjustTextFieldPosition` to  `openKeyboard`. 
But `openKeyboard` is only invoked once when keyboard is opened.

**Comment by dumganhar** [comment]
Bang!
@HoGarfield , Any updates?

**Comment by HoGarfield** [comment]
What`s the meaning of Bang?

 ------------------ 原始邮件 ------------------
  发件人: "James Chen";notifications@github.com;
 发送时间: 2014年1月1日(星期三) 晚上10:03
 收件人: "cocos2d/cocos2d-x"cocos2d-x@noreply.github.com; 
 抄送: "benz男"garfield_ho@qq.com; 
 主题: Re: [cocos2d-x] Change the place which functionadjustTextFieldPosition was called at (#4694)

Bang!
@HoGarfield , Any updates?

**Comment by dumganhar** [comment]
You took adjustTextFieldPosition to openKeyboard. 
But openKeyboard is only invoked once when keyboard is opened.

**Comment by HoGarfield** [comment]
128b9b18-9125-4ace-8d1d-df48a072407c
I add comment to openKeyborad.If it is only invoked once when keyboard is opened.It will print "openKeyboard" just once.Here is the log of testing.

468e802d-1f8c-4554-ae95-1471c644f7dc

It printed "openKeyboard" twice.

**Comment by dumganhar** [comment]
Did you test it by using TestCpp/ExtensionsTest/EditBoxTest?
If you could paste some codes, I will try to find out the issue.

**Comment by dumganhar** [comment]
I test it by commenting `top` and `middle` editbox in EditBoxTest. Only use the `bottom` editbox for testing.
Also I add log in `openKeyboard`, but it was invoked only once.

**Comment by dumganhar** [comment]
Issue http://www.cocos2d-x.org/issues/3599 was created.

**Comment by zilongshanren** [comment]
@HoGarfield I think we could put the adjustTextFieldPosition method in the visit method of EditBoxImplIOS.

**Comment by cocosQA** [comment]
Dear HoGarfield:
  We have created a new branch 'v3' to replace branch 'develop', and this PR has been moved from 'develop' branch to 'v3' branch, the new PR is https://github.com/cocos2d/cocos2d-x/pull/6470
The old 'develop' branch will be deleted soon, and this PR will be closed accordingly.

**Pull request closed without merging**